File: sum.js
#!/usr/bin/env node

const readline = require('readline');

const rl = readline.createInterface({ input: process.stdin });
rl.on('line', line => {
  const sum = line.split(' ').map(x => Number(x)).reduce((x, y) => x + y);
  console.log(sum);
  rl.close();
});

File: sum.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();
    let sum: i32 = input
        .split_whitespace()
        .map(|x| x.parse::<i32>().unwrap())
        .sum();
    println!("{}", sum);
}
